lib/cldr/helpers/math.ex

```diff
@@ -604,6 +603,7 @@ defmodule Cldr.Math do
       number
       |> to_float
       |> :math.sqrt()
+      |> to_string()
```

Comment:
I would leave the float -> binary conversion to `Decimal.new()`, but fix their typespec to include the available support for float inputs.

Reply:
[`Decimal.new` accepting `float` is soft-deprecated](https://github.com/ericmj/decimal/commit/d06fa1f2284b727df9d44e956e9a0942a858875a).  They removed it from the `@spec`, but left it in the code.  I did find out that `Decimal.from_float` is the supported way though, so I will change it to that instead of `to_string`.